tests/Unit/PointwiseFunctions/Hydro/EquationsOfState/Test_Enthalpy.cpp

```diff
@@ -159,6 +159,30 @@ void check_exact() {
         hydro::relativistic_specific_enthalpy(rho, eps, p));
     CHECK(get(rho) == approx(get(rho_from_enthalpy)));
   }
+  {
+    // Guarantee that the root find has the correct bracket by explicitly
+    // nonsensical there, so if the correct braket
```

Comment:
Please fix this sentence

Reply:
We need this merged ASAP because we are doing large simulations on Wheeler over the break. I think the sentence, while not great, is clear enough that it would be better for our science purposes to not delay this PR. Given some folks (including myself) are already on vacation and only doing this that are needed to do simulations over the break, I'd like to not block this PR on the sentence...

Reply:
sure, please fix later then

Reply:
Yep, Thanks!

Reply:
Just FYI, as a reminder: https://github.com/sxs-collaboration/spectre/issues/4537